### DISCUSSION TITLE
the usual problem with images in README.md

### DISCUSSION THREAD

**Discussion by esogeomedismo** [community/community#22588] [New to GitHub]
<p>I followed the guide to insert images into the README.md file but nothing seems to work</p>
<p>I have also read several messages similar to mine but I still can not display the images</p>
<p>the README with the broken images : <a href="https://github.com/esogeomedismo/fonZ/blob/master/README.md" title="look how you see the images" rel="nofollow noopener">my broken README</a></p>
<p>so let me try if I can see an image at least here </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/92e30799d645b0b2cdde995dfb421f4deb077331.png" data-download-href="/uploads/short-url/kXq7pzq0Vd0CshQn0Rdep9zkw01.png?dl=1" title="fonZaction2" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/44021103/181072083-ab309bd2-3e8e-4897-9f6e-fc1a6fc33407.png" alt="fonZaction2" data-base62-sha1="kXq7pzq0Vd0CshQn0Rdep9zkw01" width="690" height="417" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/92e30799d645b0b2cdde995dfb421f4deb077331_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">fonZaction2</span><span class="informations">949×574 30.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>well here I can see it</p>
<p>bad this don’t resolve anithing</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/esogeomedismo">@esogeomedismo</a>,</p>
<p>Thanks for being here! Would you mind sharing a link to your repo so we can take a look?</p>
<p>Best,</p>
<p>Andrea</p>

**Comment by Wabri** [comment, marked as answer]
<p>The only problem in your readme is the space between the title and the link of the image. [title] (link) is different from <a>title</a> because markdown can understand link only with this formatting.</p>
<p>I have open a pull request in your repository that solve your problem.</p>
<p>-Gabriele-</p>

**Comment by esogeomedismo** [comment]
<p>hi ola and thanks like like</p>
<p>the repo is in progress and maybe you already gave me an answer</p>
<p>so I reply and try some update</p>
<p>there: <a href="https://github.com/esogeomedismo/fonZ" rel="nofollow noopener">the repo</a></p>

**Comment by esogeomedismo** [comment]
<p>thanks</p>
<p>now I can view the images but I missed my first merge</p>
<p>where are the emoij?</p>
